snuba/clickhouse/columns.py

```diff
@@ -265,16 +273,16 @@ def __init__(self, columns):
             # also store it by the escaped name
             self._lookup[col.escaped] = col
```

Comment:
I left this section alone for now because I haven't taken the time to actually understand it. 

It will currently error:

```
snuba/clickhouse/columns.py:263: error: Need type annotation for '_lookup' (hint: "_lookup: Dict[<type>, <type>] = ...")
snuba/clickhouse/columns.py:264: error: Need type annotation for '_flattened' (hint: "_flattened: List[<type>] = ...")
```